### PR TITLE
Chore/add tests for blind auction

### PR DESCRIPTION
**Pull request by leventdem**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -27,5 +27,8 @@ jobs:
           tail -f fhevm.log | sed '/Starting JSON WebSocket server/ q'
 
           npm run fhevm:faucet
+          npm run fhevm:faucetBob
```
Comment (immortal-tofu) [inline .github/workflows/test.yml:30]: A possible evolution would be to wait for transaction validation before exit the command

File: codegen/templates.ts [hunk 2]
```diff
@@ -692,20 +687,6 @@ function implCustomMethods(): string {
         result = uint256(output[0]);
     }
 
-    function req(uint256 ciphertext) internal view {
```
Comment (immortal-tofu) [inline codegen/templates.ts:695]: This is not something done by @tremblaythibaultl in an other PR ?

File: lib/Impl.sol [hunk 3]
```diff
@@ -580,20 +580,6 @@ library Impl {
         result = uint256(output[0]);
     }
 
-    function req(uint256 ciphertext) internal view {
```
Comment (immortal-tofu) [inline lib/Impl.sol:583]: Same comment

**Review by immortal-tofu — APPROVED** [review]

**Comment by immortal-tofu** [comment]
Few comments but otherwise LGTM

File: codegen/templates.ts (hunk 2 above)
Comment (leventdem) [reply codegen/templates.ts:695]: yes just because of a rebase

File: .github/workflows/test.yml (hunk 1 above)
Comment (leventdem) [reply .github/workflows/test.yml:30]: if we wait in between it is generally ok

**Pull request merged**